pkg/model/project.proto

```diff
@@ -94,4 +94,8 @@ message ProjectRBACConfig {
     string admin = 1 [(validate.rules).string.min_len = 1];
     string editor = 2;
     string viewer = 3;
+
+    // (Optional) Enable this field will allow users not belonging
```

Comment:
nits: It defaults to `false` so possibly (Optional) isn't needed (some people can misunderstand `nil` allowed
```suggestion
    // Enable this field will allow users not belonging
```